### PR TITLE
refactor: change args and witness to single bytes

### PR DESCRIPTION
**Pull request by quake**
1. Change args and witness from `Vec<Bytes>` to `Bytes`
2. Add `load_script` system call

### CODE REVIEW

File: script/src/verify.rs [hunk 1]
```diff
@@ -591,23 +597,23 @@ mod tests {
 
         // Not enough cycles
         assert_error_eq(
-            verifier.verify(100).unwrap_err(),
+            verifier.verify(11).unwrap_err(),
```
Comment (keroro520) [inline script/src/verify.rs]: FYI, we should define these cycles as meaningful constant variables later.

File: script/src/verify.rs [hunk 2]
```diff
@@ -761,97 +767,21 @@ mod tests {
         );
     }
 
-    #[test]
```
Comment (keroro520) [inline script/src/verify.rs]: Why remove it?

Comment (quake) [reply script/src/verify.rs]: This unit test case is trying to run a script with low cycle, it's covered in `check_signature` line 600

File: script/src/verify.rs (hunk 1 above)
Comment (quake) [reply script/src/verify.rs]: Updated, please review again, thanks.

**Comment by quake** [comment]
bors r=doitian,xxuejie,keroro520

**Pull request merged**